ophyd/signal.py

```diff
@@ -1711,6 +1711,9 @@ def set(self, value, *, timeout=None, settle_time=None):
         --------
         Signal.set
         '''
+        if timeout is None:
```

Comment:
This sentenel should now check for `DEFAULT_WRITE_TIMEOUT`?

Reply:
Ok, now I understand this usage pattern.  `DEFAULT_WRITE_TIMEOUT` is a defined object, different from `None` but meant to show this value is not defined yet, right?

Reply:
Yeah, we need to sort out "did the user not pass _anything_ in which case we do our version of the default, otherwise just pass through to the underlying level" so we can thread `timeout=None` through to mean "no timeout".

Reply:
see c94f3ae 

This preserves the intent if user passed `timeout=None` meaning _do not timeout_.  It tries to use `self.write_timeout` but if _that_ was `None`, uses `self.timeout`.  Oy!